server/asset/btc/btc.go

```diff
@@ -1084,6 +1133,61 @@ out:
 	}
 }
 
+// estimateFee attempts to get a reasonable tx fee rates (units: atomic/(v)byte)
+// to use for the asset by checking estimate(smart)fee. That call can fail or
+// otherwise be useless on an otherwise perfectly functioning node. In that
+// case, an estimate is calculated from the median fees of the previous
+// block(s).
+func (btc *Backend) estimateFee() (satsPerB uint64, err error) {
+	if btc.cfg.DumbFeeEstimates {
+		satsPerB, err = btc.node.EstimateFee(btc.feeConfs)
+	} else {
+		satsPerB, err = btc.node.EstimateSmartFee(btc.feeConfs, &btcjson.EstimateModeConservative)
+	}
+	if err == nil && satsPerB > 0 {
+		return satsPerB, nil
+	} else if err != nil {
+		btc.log.Tracef("Estimatefee error for %s: %v", btc.name, err)
```

Comment:
Just noticing that this could show `<nil>` for `err` if `satsPerB` were just zero.  Could be confusing, but meh, it's a TRC.